sphinx-docs/screenshot.py

```diff
@@ -23,19 +23,41 @@ def _parse_focus(arg_str):
         return {'id': split_str[0], 'annotation': split_str[1]}
 
 def _parse_command(command):
-    # Return a dictionary with
-    #   action: the action type (if it's recognized)
-    #   options: a list of options
-    raise NotImplementedError()
+    """" Parses a command into action and options.
+
+    Returns a dictionary with following keys:
+       action (string): the action type (if it's recognized)
+       options (list): a list of options
+
+       Both keys have empty values if action type is not recognized.
+    """
+    # Initalize action and opttions with empty values.
+    action = ''
+    options = []
+
+    command_args = command.split(' ')
+    if command_args[0] in ('click', 'send_keys', 'submit'):
+        action = command_args[0]
+        options = command_args[-1]
+
+    return {'action': action, 'options': options}
 
 def _parse_login(username, password, submit=""):
-    # username and password are just strings, but cast submit to a bool
-    # runhandler should be "_login_handler"
-    raise NotImplementedError()
+    """" Parses a command into action and options.
+
+    Returns a dictionary with following keys:
+       username (string):  the username.
+       password (string): password.
+       submit (bool): True if login form is to be submitted, false otherwise.
+
+    runhandler should be "_login_handler"
+    """
+    submit_bool = True if submit is "submit" else False
```

Comment:
I don't think the `is` keyword is what you want to use here. `is` tests for identity rather than equality. It might work as expected for string literals, but it's safer to use `==`.